frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx

```diff
@@ -111,8 +112,18 @@ export const RecentEventsBody: React.FC<RecentEventsBodyProps> = (props) => {
     <>
       <div className="co-activity-card__recent-title">
         Recent Events
-        <DashboardCardButtonLink onClick={togglePause}>
-          {paused ? 'Unpause' : 'Pause'}
+        <DashboardCardButtonLink onClick={togglePause} className="co-activity-card__recent-actions">
+          {paused ? (
+            <>
+              <PlayIcon className="co-activity-item__icon" />
+              Unpause
```

Comment:
Not new, but should this be "Resume"? I think would match the play icon better.

@andybraren ?

Reply:
Agreed, was just thinking it felt a bit off with the new icon. 👍

Reply:
changed to `Resume`